service_management/app/controllers/analysis/SampleObjectController.scala

```diff
@@ -105,4 +105,12 @@ class SampleObjectController @Inject()(
 
     }
 
+  def getTreatmentList(mid: MuseumId) =
```

Comment:
I think this would fit better in a `TreatmentController`. Since we're likely going to add more endpoints for interacting with that type of data (?).